modules/extension/xsd/xsd-fes/src/test/java/org/geotools/filter/v2_0/bindings/FilterTypeBindingTest.java

```diff
@@ -205,9 +205,10 @@ public void testEncodeSpatial() throws Exception {
         
         Element e = getElementByQName(doc, FES.Intersects);
         assertNotNull(e);
+        print(e);
```

Comment:
Please do not print to stdout/stderr. Please log instead so output can be silenced by the build infrastructure.